docs/sbctl.8.txt

```diff
@@ -67,6 +67,12 @@ EFI signing commands
                 files and unset the immutable attribute before enrolling
                 certificates.
 
+        *--export*;;
+                Export the keys we intend to enroll as EFI Signatlure Lists
```

Comment:
```suggestion
                Export the keys we intend to enroll as EFI Signature Lists
```

Reply:
Yes, also missing a note it's exporting it into the current working directory.